requirements.txt

```diff
@@ -1,14 +1,10 @@
-pytket ~= 0.6
-pytket-qiskit ~= 0.6
-pytket-pyquil ~= 0.6
-pytket-qsharp ~= 0.6
-pytket-pyzx ~= 0.5
-pytket-projectq ~= 0.5
-pytket-cirq ~= 0.5
-pytket-honeywell ~= 0.4
-pytket-aqt ~= 0.3
-pytket-braket ~= 0.2
-pytket-qulacs ~= 0.3
-openfermion ~=0.10
-pybind11 ~= 2.5
-plotly ~= 4.8
+pytket == 0.8.0
+pytket-cirq == 0.8.0
+pytket-honeywell == 0.7.0
+pytket-projectq == 0.7.0
+pytket-pyquil == 0.8.0
+pytket-qiskit == 0.8.0
+pytket-qsharp == 0.9.0
+pytket-qulacs == 0.5.0
+
+git+git://github.com/willsimmons1465/jupyter-sphinx.git@issue-140#egg=jupyter-sphinx
```

Comment:
I thought will said this had been merged now?

Reply:
Yeah it has been merged but still not released ... :-(